doc/uftrace-record.md

```diff
@@ -512,6 +513,24 @@ all (supported) functions.  It shows two arguments for "main" and a single
 string argument for "puts".  If you simply want to see all arguments and
 return values of every functions (if supported), use -a/\--auto-args option.
 
+For example,
+
+    $ gcc -g -pg ../tests/s-autoargs.c -o abc
+    $ uftrace record -a ./abc hello
```

Comment:
You'd better make the program name same as the source name to avoid confusion.  And I assume the base directory of the source so it should be:

```
$ gcc -g -pg -o autoargs tests/s-autoargs.c
$ uftrace record -a ./autoargs hello
```